### PR TITLE
feat: Add include_protos parameter support for GAPICBazel

### PR DESCRIPTION
**Pull request by vam-google**
This is needed to migrate python libraries, which are heavily dependent on this feature.

### CODE REVIEW

**Comment by vam-google** [comment]
@busunkim96 PTAL

File: synthtool/gcp/gapic_bazel.py [hunk 1]
```diff
@@ -95,6 +96,15 @@ def _generate_code(
                 f"Unable to generate {service}, the sources repository repository"
                 "is unavailable."
             )
+        if not bazel_target or include_protos:
```
Comment (SurferJeffAtGoogle) [inline synthtool/gcp/gapic_bazel.py:99]: Add a comment describing what the next 8 lines of code are doing.

Comment (vam-google) [reply synthtool/gcp/gapic_bazel.py:101]: I refactored that section, hopefully making it easier to understand, and added the comments. PTAL.

Comment (SurferJeffAtGoogle) [reply synthtool/gcp/gapic_bazel.py:101]: Looks great.  Thanks for the changes.

**Comment by vam-google** [comment]
@SurferJeffAtGoogle @busunkim96 Can you pleaes push this?

**Pull request merged**